### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by Chike1990**
Todo List is a web application that allows users to add/remove the list of "to-dos". It provides an interface for adding to do activities, for visualizing the list of added tasks to complete. This is achieved using plain HTML, CSS, JavaScript objects, arrays, DOM, and basic events.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+- Git flow
+- GitHub
+
+### Prerequisites
+
+- Terminal (Bash)
+- Git
+- JavaScript
+
+### Setup: How to start the TicTacToe game
```
Comment (bellom) [inline README.md]: I think you might want to change the name to Todo not TicTacToe right?

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+
+- clone our repo: `https://github.com/Chike1990/ToDo_List`
+- cd into: `cd index.html`
+- run `index.html`
```
Comment (bellom) [inline README.md]: Let's try not to mislead the newbies.

I think if you follow the steps you listed you won't be able to run the app.

Obviously, you can't cd a file, we can only cd a folder.

**IMPORTANT:** You're supposed to set up [Webpack](https://webpack.js.org/guides/getting-started/#using-a-configuration) according to the project requirement.
And make sure CSS is loaded by Webpack Style/CSS Loader.

Then after successfully set up, you can now write the following steps.

```
- git clone your repo
- cd folder
- npm install
- npm start
```

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Chike1990  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 
_Kindly check the comments under the review._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Chike1990 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chike1990** [comment]
Thanks 😊

**Pull request closed without merging**